src/util/query_exec_util.cpp

```diff
@@ -21,6 +22,7 @@
 #include "settings/settings_manager.h"
 #include "transaction/transaction_context.h"
 #include "transaction/transaction_manager.h"
+#include "util/query_exec_util.h"
```

Comment:
Our convention seems always put this at the top of the file?

Reply:
Specifically if you use CLion's formatter (ctrl-alt-L) it does this, but I don't think the `clang-format` tool does this.